script/custom-eslint/enzyme-always-unmount/package.json

```diff
@@ -0,0 +1,11 @@
+{
+	"name": "eslint-plugin-enzyme-always-unmount",
```

Comment:
update to better match eslint rule naming conventions: https://eslint.org/docs/developer-guide/working-with-rules-new#rule-naming-conventions